From cc9d8b544bfec9a022650639c3d9b813d4ba9632 Mon Sep 17 00:00:00 2001
From: Naveen Saini <naveen.kumar.saini@intel.com>
Date: Thu, 16 Jul 2020 15:47:05 +0800
Subject: [PATCH] Don't build demos with questionably licensed data

Some of the models don't have open source compatible licenses:
don't build demos using those. Also don't build demos that need
resources that are not included.

ssao:
        Sibenik model, no license found

deferred:
deferredmultisampling:
deferredshadows:
        armor model, CC-BY-3.0

vulkanscene:
imgui:
shadowmapping:
        vulkanscene model, no license found

indirectdraw:
        plant model, no license found

hdr:
pbribl:
pbrtexture:
        Require external Vulkan Asset Pack

gltfloading
gltfscenerendering
gltfskinning
	glTF model, Required Additional Asset Pack

Upstream-Status: Inappropriate [configuration]

Signed-off-by: Jussi Kukkonen <jussi.kukkonen@intel.com>
Signed-off-by: Maxin B. John <maxin.john@intel.com>
Signed-off-by: Naveen Saini <naveen.kumar.saini@intel.com>
---
 examples/CMakeLists.txt | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/examples/CMakeLists.txt b/examples/CMakeLists.txt
index 8d40c6b2..7a6ae8f6 100644
--- a/examples/CMakeLists.txt
+++ b/examples/CMakeLists.txt
@@ -63,21 +63,12 @@ set(EXAMPLES
 	conditionalrender
 	conservativeraster
 	debugmarker
-	deferred
-	deferredmultisampling
-	deferredshadows
 	descriptorsets
 	displacement
 	distancefieldfonts
 	dynamicuniformbuffer
 	gears
 	geometryshader
-	gltfloading
-	gltfscenerendering
-	gltfskinning
-	hdr
-	imgui
-	indirectdraw
 	inlineuniformblocks
 	inputattachments
 	instancing
@@ -93,8 +84,6 @@ set(EXAMPLES
 	parallaxmapping
 	particlefire
 	pbrbasic
-	pbribl
-	pbrtexture
 	pipelines
 	pipelinestatistics
 	pushconstants
@@ -102,12 +91,8 @@ set(EXAMPLES
 	radialblur
 	renderheadless
 	screenshot
-	shadowmapping
-	shadowmappingomni
-	shadowmappingcascade
 	specializationconstants
 	sphericalenvmapping
-	ssao
 	stencilbuffer
 	subpasses
 	terraintessellation
@@ -121,7 +106,6 @@ set(EXAMPLES
 	texturesparseresidency
 	triangle
 	viewportarray
-	vulkanscene
 )
 
 buildExamples()
-- 
2.17.1

